userguide/advanceduse/reverse-tethering.rst

```diff
@@ -1,34 +1,76 @@
 Reverse tethering
 =================
 
-This tutorial helps you to connect your Ubuntu Touch device to your computer to access the Internet on it.
+This tutorial helps you to connect your Ubuntu Touch device to your computer to access the internet on it.
 This is useful if there is no available Wi-Fi connection or you don't have a data subscription on your Ubuntu Touch device.
 
-Prerequisite: A Ubuntu Touch device with "Developer mode" on, attached via USB to the computer connected to the Internet.
+Prerequisite: A Ubuntu Touch device with "Developer mode" on, attached via USB to the computer connected to the internet.
+
+Tested: 2023-01-02 with Ubuntu Touch 16.04 (OTA-24) and Ubuntu 22.04.
 
 Steps
 -----
 
-1. On the Ubuntu Touch device: ``android-gadget-service enable rndis``
+1. On the Ubuntu Touch device:
+
+  - Run: ``android-gadget-service enable rndis``
+
+  - Reboot the phone.
+
+2. On your computer:
 
-2. On your computer: get your RNDIS IP address e.g:``hostname -I``
+  - Get your RNDIS IP address:
 
-3. On the Ubuntu Touch device: 
+    Run: ``ip a``
 
-      add a gateway: ``sudo route add default gw YOUR_COMPUTER_RNDIS_IP``
+    Look for an interface named ``usb0`` in the output.
 
-      add nameservers: ``echo "nameserver 8.8.8.8" | sudo tee /etc/resolv.conf``
+    For example::
```

Comment:
I find the formatting strange. Consider either making it a bullet list or make it a paragraph without line breaks

Reply:
I removed the `For example` headings.